File: exercises/small_problems_101/easy_9/welcome_stranger.rb
def greetings(name, title)
  full_name = name.join(' ')
  full_title = title.values.join(' ')
  puts "Hello, #{full_name} to have a #{full_title} around."
end

greetings(['John', 'Q', 'Doe'], { title: 'Master', occupation: 'Plumber' })

File: frontend/javascript/objects/welcome_stranger.js
function greetings(array, object) {
  var fullName = array.join(' ');
  console.log('Hello, ' + fullName + '! ' + 'Nice to have a ' + object.title + ' ' + object.occupation + ' around.');
}

greetings(['John', 'Q', 'Doe'], { title: 'Master', occupation: 'Plumber' });